docs/docs/training-data-format.mdx

```diff
@@ -1,355 +1,655 @@
 
-Lookup tables provide a convenient way to supply a list of entity examples.
-The supplied lookup table files must be in a newline-delimited format.
-For example, `data/test/lookup_tables/plates.txt` may contain:
+Lookup tables provide a convenient way to supply a list of entity
+examples. The format is as follows:
 
-```python (docs/sources/data/test/lookup_tables/plates.txt)
+```yaml
+- lookup: banks
+  examples: |
+  - JPMC
+  - Bank of America
 ```
 
-And can be loaded and used as shown here:
+The name of the lookup table is subject to the same constraints as the
+name of a regex feature.
 
-```md
-## lookup:plates
-data/test/lookup_tables/plates.txt
-
-## intent:food_request
-- I'd like beef [tacos](plates) and a [burrito](plates)
-- How about some [mapo tofu](plates)
-```
+When you supply a lookup table in your training data, the contents of that table
+are combined into one large regular expression. This regex is used to check
+each training example to see if it contains matches for entries in the
+lookup table.
 
-When lookup tables are supplied in training data, the contents are combined
-into a large regex pattern that looks for exact matches in
-the training examples. These regexes match over multiple tokens, so
-`lettuce wrap` would match `get me a lettuce wrap ASAP` as `[0 0 0 1 1 0]`.
-These regexes are processed identically to the regular regex patterns
-directly specified in the training data.
+Lookup table regexes are processed identically to the regular
+regex patterns directly specified in the training data and can be used
+either with the [RegexFeaturizer](../nlu/components#regexfeaturizer)
+or with the [RegexEntityExtractor](../nlu/components#regexentityextractor).
 
 :::note
-If you are using lookup tables in combination with [RegexFeaturizer](./components/featurizers.mdx#regexfeaturizer), there must be a few examples of matches
-in your training data. Otherwise the model will not learn to use the lookup table match features.
 
+If you are using lookup tables in combination with the `RegexFeaturizer`, there must be a few examples of matches  
+in your training data. Otherwise the model will not learn to use the
+lookup table match features.
 :::
 
-:::caution
-You have to be careful when you add data to the lookup table.
-For example if there are false positives or other noise in the table,
+:::warning
+
+You have to be careful when you add data to the lookup table.  
+For example, if there are false positives or other noise in the table,
 this can hurt performance. So make sure your lookup tables contain
 clean data.
-
 :::
 
-## Normalizing Data
 
-<a aria-hidden="true" tabIndex="-1" className="anchor enhancedAnchor" id="entity-synonyms"></a>
+## Conversation Training Data
+
+Stories and rules are both representations of conversations between a user and a conversational
+assistant. They are used to train the dialogue management
+model. **Stories** are used to train a machine learning model to identify patterns
```

Comment:
just a note -  the stories page hasn't been updated yet, it still shows markdown training data. We can link so long though
```suggestion
model. [**Stories**](stories.mdx) are used to train a machine learning model to identify patterns
```